frontend/src/pages/HomePage.tsx

```diff
@@ -0,0 +1,30 @@
+import landingImage from '../assets/landing.png';
+import appDownloadImage from '../assets/appDownload.png';
+
+const HomePage = () => {
+  return (
+    <div className="flex flex-col gap-12">
+      <div className="md:px-32 bg-white rounded-lg shadow-md py-8 flex flex-col gap-5 text-center -mt-16">
+        <h1 className="text-5xl font-bold tracking-tight text-orange-600">
+          Tuck into a takeaway today
+        </h1>
+        <span className="text-xl">Food is just a click away!</span>
+      </div>
+      <div className="grid md:grid-cols-2 gap-5">
+        <img src={landingImage} />
+        <div className="flex flex-col items-center justify-center gap-4 text-center">
+          <span className="font-bold text-3xl tracking-tighter">
+            Order takeaway even faster!
+          </span>
+          <span>
+            Download the Eats App for faster ordering and personalized
+            recommendations
+          </span>
+          <img src={appDownloadImage} />
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/11#issuecomment-2358307757>#f60478</a><span> - 09/18/2024, 12:19 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The &lt;img&gt; element for the app download image lacks an alt attribute, which is essential for accessibility. A similar issue was also found in frontend/src/pages/HomePage.tsx (line 14).
</div>
<div id="fix">
<b>Fix:</b> Add an appropriate alt attribute to the &lt;img&gt; element describing the app download image.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -23 +23 @@
 -          <img src={appDownloadImage} />
 +          <img src={appDownloadImage} alt="Download our app" />
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged